tools/htmlpurifier/HTMLPurifier.standalone.php

```diff
@@ -13178,7 +13178,7 @@ public function validate($aIP, $config, $context)
             }
 
             while (count($first) < 8) {
-                array_push($first, '0');
+                $first[] = '0';
```

Comment:
Although I agree with the content of this PR, I'd prefer to leave this file untouched as we deal with an external lib.

Reply:
As we use it directly, it's not an issue (doing this, it's like we have a fork).

Can we plan to use the composer dependency instead (https://github.com/ezyang/htmlpurifier)?

Reply:
As HTML Purifier is a third party module it's better use via composer.

http://htmlpurifier.org/download#Composer

https://github.com/ezyang/htmlpurifier

Reply:
Sure, but we need to plan it in another contribution (and check we didn't change anything in it, too :/ )

Reply:
It seems we already altered the lib, which is an issue :/